### DISCUSSION TITLE
Load global css in plugin

### DISCUSSION THREAD

**Discussion by k0mar12** [nuxt/nuxt#15887] [Questions]
Good day.

Is it possible to load global css in plugin?
I need for admin side, lets say, **tailwind** but for client side - **bulma**.
If i will just inject in **nuxt.config** then, i will get both of them in one time, but i dont really need it.
I`m looking for some solution in plugin, like if some condition passed, then load global css.

Thanks

**Comment by ndragun92** [comment]
I guess u can do normal JS way via plugin and append script to the head or you can make leverage layout and do it with useHead?

**Comment by Baroshem** [comment]
What about creating a module instead of a plugin and add CSS similarly to how Tailwind module was created? https://github.com/nuxt-modules/tailwindcss/tree/main/src or https://github.com/stylify/packages/blob/master/packages/nuxt/src/module.ts

**Reply by k0mar12** [reply]
Sounds interesting, i will check it, thanks

**Reply by Machy8** [reply]
@k0mar12 The part of the plugin that pushes the CSS into the Nuxt is this one https://github.com/stylify/packages/blob/master/packages/nuxt/src/module.ts#L122-L132.

If you plan to split the CSS into multiple files or use multiple frameworks at once, you will probably need to use CSS layers. For some reason, the CSS order in Nuxt (especially in version 2, I haven't tested it extra yet for the 3 version) sometimes changes and therefore utilities (and CSS commonly) are overridden incorrectly. It should not happen if it will be loaded for admin/front layout separately as one file, but just in case I have mentioned it. This problem gave me a headache, so I hope you will not have to solve it.
